client/_attachments/landing_after_login_ukr.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="mock-ups/bootstrap/css/bootstrap.css"/>
+    <link rel="stylesheet" href="mock-ups/bootstrap/css/bootstrap-theme.css"/>
+    <link rel="stylesheet" href="mock-ups/lib/font-awesome-4.4.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="mock-ups/styleNew.css"/>
+    <title>landing after login</title>
+</head>

```

Comment:
There should not be html->head+body markup, only body contents(similar to landing_after_login_ru.html)